lib/specinfra/backend/docker.rb

```diff
@@ -12,7 +12,9 @@ def initialize
       if image = Specinfra.configuration.docker_image
         @images = []
         @base_image = ::Docker::Image.get(image)
-        create_and_start_container
+
+        env = Specinfra.configuration.docker_env || {}
```

Comment:
I assumed that env was for the running context as opposed to inside the docker container. Ie DOCKER_URL would live in Specinfra.configuration.env, these are for inside the container. 


If that is not a valid distinction then I'll patch it up.

On Sat, Feb 28, 2015 at 12:28 PM, Gosuke Miyashita
<notifications@github.com> wrote:

>> @@ -12,7 +12,9 @@ def initialize
>>        if image = Specinfra.configuration.docker_image
>>          @images = []
>>          @base_image = ::Docker::Image.get(image)
>> -        create_and_start_container
>> +
>> +        env = Specinfra.configuration.docker_env || {}
> Why don't you use `Specinfra.configuration.env` ?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/serverspec/specinfra/pull/328/files#r25551917